Resources/public/js/views/fields/ez-date-editview.js

```diff
@@ -103,12 +325,46 @@ YUI.add('ez-date-editview', function (Y) {
          * @return {Number}
          */
         _getFieldValue: function () {
-            var valueOfInput = this._getInputNode().get('valueAsNumber');
-            if (valueOfInput) {
-                return {timestamp: valueOfInput/1000};
-            }
-            return null;
+            var valueOfTextInput = Y.Date.parse(this._getInputNode().get('value'));
```

Comment:
In https://github.com/ezsystems/PlatformUIBundle/pull/146#discussion_r22382808, I suggested to use the `date` attribute instead of requesting the DOM here, is there an issue with that or just an oversight ?